### PR TITLE
LRQA-50761 | master

### PR DESCRIPTION
**Pull request by melodywu**
https://issues.liferay.com/browse/LRQA-50761

### CODE REVIEW

**Comment by melodywu** [comment]
ci:test:sf

**Comment by melodywu** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase [hunk 1]
```diff
@@ -918,6 +918,85 @@ definition {
...
+	test CanReactivateNotCompletedDeletedUser {
+		User.logoutAndLoginPG(
+			userLoginEmailAddress = "userea@liferay.com",
+			userLoginFullName = "userfn userln");
+
+		DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "guest");
+
+		DMFolder.addCP(dmFolderName = "Folder1");
+
+		DMDocument.addMultipleCP(dmDocumentFileList = "Document_1.jpeg,Document_2.docx");
```
Comment (patriciajeanperez) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase:932]: @melodywu why was Document_1.jpeg added if we didn't have to use it in this test?

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase [hunk 2]
```diff
@@ -918,6 +918,85 @@ definition {
 
 	}
 
+	@description = "This test deleted personal data that has not been completed can be reactivated."
+	@priority = "3"
+	test CanReactivateNotCompletedDeletedUser {
```
Comment (patriciajeanperez) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase:923]: @melodywu Description and title needs rewording. Test is asserting that entries are still anonymous after activating user.

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase [hunk 3]
```diff
@@ -918,6 +918,85 @@ definition {
...
+		Click.clickAt(locator1 = "Icon#INFO");
+
+		AssertTextPresent(
+			locator1 = "Sidebar#SIDEBAR",
+			value1 = "Anonymous Anonymous");
+
+		ApplicationsMenu.gotoPortlet(
+			category = "Users",
+			panel = "Control Panel",
+			portlet = "Users and Organizations");
```
Comment (patriciajeanperez) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase:973]: @melodywu can open Users Admin via url

**Review by patriciajeanperez — COMMENTED** [review]
Hi @melodywu! Good job. Please check some minor comments. It's also a very long test that will benefit from separating the macros into tasks for readability. Thanks!

**Comment by melodywu** [comment]
ci:test:sf

**Comment by melodywu** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase [hunk 4]
```diff
@@ -918,6 +918,84 @@ definition {
 
 	}
 
+	@description = "Test is asserting that entries are still anonymous after activating user."
+	@priority = "3"
+	test CanReactivateNotCompletedDeletedUser {
```
Comment (patriciajeanperez) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase:923]: @melodywu can change title to "EntriesAreStillAnonymousAfterActivatingUser"

Comment (melodywu) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase:923]: Done

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase (hunk 1 above)
Comment (melodywu) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase]: Done

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase (hunk 2 above)
Comment (melodywu) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase]: Done

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase (hunk 3 above)
Comment (melodywu) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase]: Done

**Comment by melodywu** [comment]
ci:test:sf

**Comment by melodywu** [comment]
ci:test:relevant

**Comment by patriciajeanperez** [comment]
ci:forward

**Comment by patriciajeanperez** [comment]
ci:forward

**Pull request closed without merging**